src/components/activities/adminActivity/SideMenu/SideMenu.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FunctionComponent, ReactNode, useCallback } from "react";
+import {
+  AirlineSeatFlat,
+  AssignmentInd,
+  BlurCircular,
+  ExpandMoreSharp,
+  GroupWork,
+  Healing,
+  LocalDrink,
+  LocalHospitalSharp,
+  People,
+  SupervisedUserCircle,
+} from "@material-ui/icons";
+import { IAdminSection } from "../types";
+import { useTranslation } from "react-i18next";
+import { useNavigate } from "react-router";
+import classes from "./SideMenu.module.scss";
+import { MenuItem } from "../../../accessories/menuItem";
+
+interface IOwnProps {
+  setAdminSection: React.Dispatch<React.SetStateAction<IAdminSection>>;
+  adminSection: IAdminSection;
+}
+
+const SideMenu: FunctionComponent<IOwnProps> = ({
+  setAdminSection: setUserSection,
+  adminSection: userSection,
+}) => {
+  const { t } = useTranslation();
+
+  const navigate = useNavigate();
+
+  const changeAdminSection = useCallback(
+    (section: IAdminSection) => {
+      setUserSection(section);
+      navigate(`${section}`, { replace: true });
+    },
+    [navigate, setUserSection]
+  );
+
+  const menuItems: { key: IAdminSection; icon: ReactNode }[] = [
+    {
+      key: "wards",
+      icon: <AirlineSeatFlat />,
+    },
+    {
+      key: "diseases",
+      icon: <BlurCircular />,
+    },
+    {
+      key: "exams",
+      icon: <AssignmentInd />,
+    },
+    {
+      key: "operations",
+      icon: <Healing />,
+    },
+    {
+      key: "vaccines",
+      icon: <LocalDrink />,
+    },
+    {
+      key: "suppliers",
+      icon: <SupervisedUserCircle />,
+    },
+    {
+      key: "usersGroups",
+      icon: <People />,
+    },
+    {
+      key: "manageTypes",
+      icon: <GroupWork />,
+    },
+  ];
+
+  return (
+    <div data-cy="admin-side-menu" className={classes.menu}>
+      {menuItems.map((item) => (
+        <MenuItem
+          key={item.key}
+          icon={item.icon}
+          label={t(`nav.${item.key}`)}
+          selected={userSection === item.key}
+          onClick={() => {
+            changeAdminSection(item.key);
+          }}
+        />
+      ))}
+      <span className={classes.label}>{t("nav.hospital")}</span>
+      <MenuItem
+        icon={<LocalHospitalSharp />}
+        label={t(`nav.hospitalInfo`)}
+        trailingIcon={<ExpandMoreSharp />}
+        onClick={() => {}}
+      />
```

Comment:
Why bother creating a different level of navigation ? Flattening those may enhance readability
<img width="1406" alt="Screenshot 2024-04-21 at 08 32 18" src="https://github.com/informatici/openhospital-ui/assets/597067/d063b415-28b8-4518-83e0-cabf1e1397a8">

Reply:
Hello @gasp ! Thanks for your reviews.

Reply:
@gasp , please let me know  or mark this one as resolve.

Reply:
Hi @Steve thanks for answering. Where can I find the Mockup ?